AirCasting/SDSync/SDSyncFileWritingService.swift

```diff
@@ -37,11 +37,8 @@ final class SDSyncFileWritingService: SDSyncFileWriter {
             }
             return []
         }
-        let toReturn = SDCardSessionType.allCases.compactMap { type -> (URL, SDCardSessionType)? in
-            let url = fileURL(for: type)
-            guard fileHandles[url] != nil else { return nil }
-            return (url, type)
-        }
+        
+        let toReturn = [(fileURL(for: .mobile), SDCardSessionType.mobile), (fileURL(for: .fixed), SDCardSessionType.fixed)]
```

Comment:
this part is a bit sketchy (I wanted to fix it by introducing 3rd file) - `SDCardSessoinType` has 3 values, we only return two here and the knowledge that mobile sessions are merged with cellular ones is very hidden and implicit. We need to refactor this system to operate on 3 files (todo for next version I guess)

Reply:
Actually fixed are merged with cellular. I know, it's kind of hidden. I will find out why android team did it this way. I guess it's because we are sending fixed and cellular together to backend.